pages/common/lldb.md

```diff
@@ -0,0 +1,15 @@
+# lldb
+
+> The LLDB Debugger.
+
+- Debug an executable:
+
+`lldb {{executable}}`
+
+- Attach lldb to running process with id {{PID}}:
+
+`lldb -p {{PID}}`
+
+- Wait for process to launch and attach to it:
+
+`lldb -w -n {{name}}`
```

Comment:
Got it. In that case, lets change the description to -
`Wait for a new process to launch with the given name and attach to it:`